backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/invoice/InvoicesRestController.java

```diff
@@ -312,8 +317,28 @@ public ResponseEntity<JsonCreateInvoiceResponse> createInvoice(@NonNull @Request
 
 			return ResponseEntity.unprocessableEntity()
 					.body(JsonCreateInvoiceResponse.builder()
-								  .errors(ImmutableList.of(JsonErrors.ofThrowable(ex, adLanguage)))
-								  .build());
+							.errors(ImmutableList.of(JsonErrors.ofThrowable(ex, adLanguage)))
+							.build());
+		}
+	}
+
+	@PostMapping("/docReview")
+	public ResponseEntity<JsonCreateInvoiceReviewResponse> createReview(@NonNull @RequestBody final JsonInvoiceReviewUpsertItem jsonInvoiceReviewUpsertItem)
+	{
+		try
+		{
+			final Optional<JsonCreateInvoiceReviewResponse> response = jsonInvoiceReviewService.upsert(jsonInvoiceReviewUpsertItem);
```

Comment:
it's a matter of taste, but i think u could inline the `response`.

Reply:
inlined.